src/subcommand/giraffe_main.cpp

```diff
@@ -350,12 +350,14 @@ void help_giraffe(char** argv) {
     << "  -s, --cluster-score INT       only extend clusters if they are within INT of the best score [50]" << endl
     << "  -S, --pad-cluster-score INT   also extend clusters within INT of above threshold to get a second-best cluster [0]" << endl
     << "  -u, --cluster-coverage FLOAT  only extend clusters if they are within FLOAT of the best read coverage [0.3]" << endl
+    << "  -U, --max-min INT             use at maximum INT unique non-overlapping minimizers [1000]" << endl
```

Comment:
When you don't pass `--exclude-overlapping-min`, this is just a limit on the maximum number of minimizers overall, right?

Reply:
```suggestion
    << "  -U, --max-min INT             use at most INT minimizers [500]" << endl
```
The default in the code is 500 for the argument value.